zeppelin-web/src/app/app.controller.js

```diff
@@ -45,4 +47,42 @@ angular.module('zeppelinWebApp').controller('MainCtrl', function($scope, $rootSc
   BootstrapDialog.defaultOptions.onshown = function() {
     angular.element('#' + this.id).find('.btn:last').focus();
   };
+
+  $rootScope.$on('hasNewStatus', function(event, data) {
+    if (!event.defaultPrevented && data && data === true && $rootScope.hasNewStatus === false) {
+      $rootScope.hasNewStatus = true;
+      PageTitleNotification.On('You have a job finished!!!', 1000);
+      event.preventDefault();
+    }
+  });
+
+  // Blinking page title for finished job notification
+  $window.onblur = function (){
+    $rootScope.windowFocus = false;
+  };
+
+  $window.onfocus = function (){
+    $rootScope.windowFocus = true;
+    if($rootScope.hasNewStatus === true) {
+      $rootScope.hasNewStatus = false;
+      PageTitleNotification.Off();
+    }
+  };
+
+  var PageTitleNotification = {
```

Comment:
by convention, JS objects and props are `camelCased`, (e.g. `PageTitleNotification -> pageTitleNotification`, `Vars -> vars`)